app/views/case_contacts/form/expenses.html.erb

```diff
@@ -97,6 +97,12 @@
     <% end %>
 
     <div class="actions mb-10 case-contacts-form-buttons">
+      <div class="case-contacts-form-options checkbox-style flex-grow-1">
+        <%= form.fields_for :metadata do |metadata_form| %>
+          <%= metadata_form.check_box :create_another, class: 'form-check-input ' %>
+          <%= metadata_form.label :create_another,  "Create Another", class: 'form-check-label ' %>
+        <% end %>
+      </div>
```

Comment:
I am a bit worried users might not notice this button, can we slide it all the way right next to the submit button?

Reply:
![Screenshot 2024-08-04 at 12 07 56 AM](https://github.com/user-attachments/assets/c3754970-26f4-4ede-971d-214f163a3952)